examples/httpserver/bin/hello_world_server.dart

```diff
@@ -18,9 +18,8 @@ Future main() async {
 
   // #docregion listen
   await for (HttpRequest request in server) {
-    request.response
-      ..write('Hello, world!')
-      ..close();
+    request.response.write('Hello, world!');
```

Comment:
Do these *_.server.dart examples need to be kept in sync with their counterparts in `dart-lang/dart-tutorials-samples`? Not a blocker for this PR, I'm mostly asking just to get a general sense of our approach to this right now.

See, e.g. https://github.com/dart-lang/dart-tutorials-samples/blob/master/httpserver/bin/hello_world_server.dart
(There are very slight differences between the code for these examples between dart-lang/site-www and dart-lang/dart-tutorials-samples)

Reply:
I'll create a PR for dart-lang/dart-tutorials-samples repo to bring those samples up to date. I see anagram, for example, is stuck in Dart v1.

Reply:
That took slightly longer than expected.

https://github.com/dart-lang/dart-tutorials-samples/pull/49